Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -214,6 +214,46 @@ Create the object store resources:
 If encryption or compression on the wire is needed, specify the `--v2-port-enable` flag.
 If the v2 address type is present in the `ceph quorum_status`, then the output of 'ceph mon data' i.e, `ROOK_EXTERNAL_CEPH_MON_DATA` will use the v2 port(`3300`).
 
+### NFS storage
```

Comment:
I wonder if we should include a section on external ceph in the main nfs.md doc, then include a link to that section from this doc. The export creation seems like a duplicate from that doc as well. @BlaineEXE thoughts?

Reply:
Oops. Yes, I would prefer to document this in the NFS section and link to it from here.

Reply:
Updated